ingest/template/vanilla_spreadsheet_builder.py

```diff
@@ -69,11 +77,13 @@ def build(self, spreadsheet_tabs_template: SchemaTemplate):
                         worksheet.set_row(3, None, None, {'hidden': True})
 
                     if column_index == 0:
-                        worksheet.set_row(0, 30)
-                        worksheet.set_row(4, 30)
-                        worksheet.write(4, column_index, "FILL OUT INFORMATION BELOW THIS ROW", self.header_format)
+                        worksheet.set_row(0, self.header_row_height)
+                        worksheet.set_row(1, self.header_row_height)
+                        worksheet.set_row(2, self.header_row_height)
+                        worksheet.set_row(4, self.guide_row_height)
+                        worksheet.write(4, column_index, "FILL OUT INFORMATION BELOW THIS ROW", self.guide_format)
                     else:
-                        worksheet.write(4, column_index, '', self.header_format)
+                        worksheet.write(4, column_index, '', self.guide_format)
 
```

Comment:
@aaclan-ebi is this what you meant? I defined variables within the base SpreadsheetBuilder object and use the variables instead of numbers. I don't think we have an idea of a config for the spreadsheets

Reply:
I was just thinking if you have more time, there could be a row config dictionary defined on top to organise those fixed values.

```
ROW_CONFIG = {
	'header': {
		'row_index': '0',
		'height': '50',
		'format': {}
	},
	'guide': {
		'row_index': '4',
		'height': '30',
		'format': {}
	},
	'description': {
		'row_index': '4',
		'height': '30'
	}
}
```

but i'm already happy with just variables

Reply:
I imagine it'll look something like this:

https://github.com/ebi-ait/ingest-client/blob/master/ingest/importer/spreadsheet/ingest_worksheet.py#L7-L9

could put them on top to easily set them if we need to , dunno if this is the best practice but it makes sense to me